docs/DeveloperGuide.md

```diff
@@ -265,8 +284,14 @@ The following sequence diagram shows how the `sort` command works:
 * Alternative 3 : `sort` command takes in a value so the user can specify how many profiles they wish to view
     * Pros: Provides maximum flexibility and functionality
 
+### `remark`
+The `remark` command allows the user to add optional remarks / comments to a person.
 
-
+#### Implementation:
+`remark` works similarly to `edit`, but is only editing the `Remark` attribute of the `Person`. 
+To add this feature, the following were done:
+- Include optional `Remark` attribute in `Person` class
+- Edit existing features to parse remarks (e.g. in `add` and `edit` to have a new CLI flag)
```

Comment:
Had some merge conflicts previously as the edited part was the same as Rohan's edited part of the file. Hence, I've moved your section to below Rohan's one to resolve it.